### PR TITLE
Update cassandra tutorial

### PR DESCRIPTION
**Pull request by aledbf**
Sync tutorial content with `kubernetes/examples`

This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/website/7478)

### CODE REVIEW

File: docs/tutorials/stateful-application/cassandra.md [hunk 1]
```diff
@@ -11,9 +11,9 @@ Deploying stateful distributed applications, like Cassandra, within a clustered
 
 **Cassandra Docker**
 
-The Pods use the [`gcr.io/google-samples/cassandra:v12`](https://github.com/kubernetes/examples/blob/master/cassandra/image/Dockerfile)
+The Pods use the [`gcr.io/google-samples/cassandra:v13`](https://github.com/kubernetes/examples/blob/master/cassandra/image/Dockerfile)
 image from Google's [container registry](https://cloud.google.com/container-registry/docs/).
-The docker is based on `debian:jessie` and includes OpenJDK 8. This image includes a standard Cassandra installation from the Apache Debian repo.  By using environment variables you can change values that are inserted into `cassandra.yaml`.
+The docker is based on [debian-base](https://github.com/kubernetes/kubernetes/tree/master/build/debian-base) and includes OpenJDK 8. This image includes a standard Cassandra installation from the Apache Debian repo.  By using environment variables you can change values that are inserted into `cassandra.yaml`.
```
Comment (bradtopol) [inline docs/tutorials/stateful-application/cassandra.md]: "The docker" seems awkwardly phrased. Perhap instead say "The docker utilized"

Comment (ahmetb) [reply docs/tutorials/stateful-application/cassandra.md]: Just say: The docker image above is based on

**Comment by ahmetb** [comment]
/lgtm
except minor nits

**Pull request merged**